src/System.Buffers/tests/ArrayPool/CollectionTests.cs

```diff
@@ -144,7 +143,6 @@ private static bool ValidateTrimState(object pool, string trimString)
         [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsPreciseGcSupported))]
         [InlineData(true)]
         [InlineData(false)]
-        [ActiveIssue(29866, TargetFrameworkMonikers.Uap)]
```

Comment:
Let's make sure to close all these issues.

Reply:
Will do with a git diff grep when this is merged :)